Xamarin/Droid/Screens/LoginDialog.cs

```diff
@@ -48,15 +58,16 @@ public override Dialog OnCreateDialog(Bundle savedInstanceState)
                 return null;
             }
 
-            serverAddressEntry = dialogView.FindViewById<EditText>(Resource.Id.serverIPEntry);
-            usernameEntry = dialogView.FindViewById<EditText>(Resource.Id.usernameEntry);
-            passwordEntry = dialogView.FindViewById<EditText>(Resource.Id.passwordEntry);
-            serverAddressEntry.Text = DrawXSettingsManager.Settings.ServerIP;
-            usernameEntry.Text = DrawXSettingsManager.Settings.Username;
+            _serverAddressEntry = dialogView.FindViewById<EditText>(Resource.Id.serverIPEntry);
```

Comment:
This is an example of the kind of stylistic renaming I would prefer to see in a separate PR because it reduces the visual noise dramatically when reviewing functional changes.